runnc-cont/config.go

```diff
@@ -0,0 +1,46 @@
+package runnc_cont
+
+// Config configuration to create a runnc-cont
+type Config struct {
+	// path to 'nabla-run' binary.
```

Comment:
Let's add the gostyle comments so they appear nice in docs since you've already made the effort to put in each comment!

`// NablaRunBin is the path to 'nabla-run' binary`

Reply:
make sense